src/Administration/Resources/app/administration/src/module/sw-customer/snippet/de-DE.json

```
@@ -223,6 +223,7 @@
       "buttonEdit": "Bearbeiten",
       "tabGeneral": "Allgemein",
       "tabAddresses": "Adressen",
+      "messageSaveInfoNumberChange": "Der Kunde hat die neue Kundennummer '{newCustomerNumber}' reserviert bekommen. Vorher: '{oldCustomerNumber}'.",
```

```suggestion
      "messageSaveInfoNumberChange": "Der Kunde hat die neu generierte Kundennummer '{newCustomerNumber}' bekommen. Vorher: '{oldCustomerNumber}'.",
```
is more clearer IMHO, and also in line with the english translation. Also maybe a hint on why this is happening may make sense, something like:
```
Due to multiple simultaneous registration, the customer got a newly generated customer number.
```

What do you think?

---

I know my text was more technical. On the other side your text is also a bit too narrowed down to registration. Would you count a customer import via API a "registration" for an admin user?

---

Please review new texts

---

LGTM :+1: